packages/core/scripts/local/DeployEMP.js

```diff
@@ -19,34 +19,29 @@
  * - `./PushPriceEMP.js`: "resolves" a pending mock oracle price request with a price.
  *
  *
- * Example: $(npm bin)/truffle exec ./scripts/local/DeployEMP.js --network test --test true --identifier ETH/BTC
+ * Example: $(npm bin)/truffle exec ./scripts/local/DeployEMP.js --network test --test true --identifier ETH/BTC --cversion 1.1.0
  */
 const { toWei, utf8ToHex, hexToUtf8 } = web3.utils;
 const { interfaceName } = require("@uma/common");
+const { getAbi, getTruffleContract } = require("../../index");
+const argv = require("minimist")(process.argv.slice(), {
+  boolean: ["test"],
+  string: ["identifier", "collateral", "cversion"]
```

Comment:
Used `cversion` (as in contract version) because I didn't want outer layers of node/yarn processes to get confused by the arg `version`.